bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -15,20 +15,21 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 package com.adobe.cq.wcm.core.components.internal.models.v3;
 
+import java.awt.*;
```

Comment:
Please list all classes

Reply:
`java.awt` is the exception from the rule. This is the fragment from the wiki page about code conventions (https://wiki.corp.adobe.com/display/tsg/Code+Conventions+For+Java) .
`The order of the package imports is:
	1.	java.*
	2.	_blank line_
	3.	javax.*
	4.	_blank line_
	5.	org.*
	6.	_blank line_
	7.	*
	8.	_blank line_
	9.	import static *
Each group of imports is sorted alphabetically. Never use wildcard imports (import java.util.*) except for java.awt`